go.mod

```diff
@@ -0,0 +1,21 @@
+module github.com/Backblaze/blazer
+
+go 1.18
+
+require (
+	github.com/golang/protobuf v1.5.3
+	github.com/google/subcommands v1.2.0
+	github.com/google/uuid v1.3.1
+	github.com/grpc-ecosystem/grpc-gateway v1.16.0
+	golang.org/x/net v0.17.0
+	google.golang.org/genproto/googleapis/api v0.0.0-20231009173412-8bfb1ae86b6c
+	google.golang.org/grpc v1.58.3
```

Comment:
The version for `grpc` should be downgraded, as trying to run `go vet ./...` with Go 1.18 triggers the following error:
```
# google.golang.org/grpc
/home/mike/.go/pkg/mod/google.golang.org/grpc@v1.58.3/server.go:2096:14: undefined: atomic.Int64
note: module requires Go 1.19
```

When testing locally, changing it to `1.58.2` fixes the issue.

Reply:
Thanks, @adamantike! @tzeejay could you change `grpc` version to `1.58.2`, retest at your end, and submit that change? Thanks!

Reply:
@adamantike hey I was going to suggest that we upgrade the Go version but I saw that you specifically set yours to `1.18` and you already did a local test so no sweat. I will give this a shot myself real quick and push another commit. Thank you for the notes

Reply:
Thanks, @tzeejay. I also wasn't able to get `go vet ./...` to report an issue with go 1.18, but I do see `go 1.19` in the `go.mod` file for `google.golang.org/grpc@v1.58.3`, so I think this is the correct way to go.